tests/unittest.cpp

```diff
@@ -1,1290 +1,1247 @@
-//#define CROW_ENABLE_LOGGING
-#define CROW_LOG_LEVEL 0
-#define CROW_ENABLE_DEBUG
```

Comment:
This seems to remove a bunch of hits on the logging code, we might need some logging tests to replace them

Reply:
I've just disabled logging in test - I think that this isn't necessary in tests. What do you think?

Reply:
It might be useful to know what the app is doing if something fails, it's why I added the `-V` tag to `ctest`, I think it's best to keep it.